### ISSUE TITLE
Handle multiple comma-separated file names for long descriptions

### ISSUE THREAD

**Issue by sjlongland** [diazona/setuptools-pyproject-migration#158]
As per the `TODO` comment in https://github.com/diazona/setuptools-pyproject-migration/pull/156/files#diff-31fbb764b62f2a2ed3f664b41f68cd6c5e5b9c8881d4a581bdab591202340ba3R145

**Comment by diazona** [comment]
Good call, thanks! And this might be a good thing to add an xfail-ing test for, even before we get around to actually fixing it.
